python/tvm/relax/frontend/nn/op.py

```diff
@@ -50,7 +50,8 @@ def _wrap_nested(expr: rx.Expr, name: str) -> Union[Tensor, Tuple[Tensor, ...]]:
     result : Union[Tensor, Tuple[Tensor]]
         The computed result.
     """
-    expr = BlockBuilder.current().emit(expr, name)
+    if not isinstance(expr, rx.DataflowVar):
```

Comment:
i believe you can just do https://github.com/apache/tvm/blob/unity/python/tvm/relax/block_builder.py#L632